docs/index.md

```diff
@@ -662,6 +662,8 @@ describe('outer', function() {
 });
 ```
 
+Skipping a test within an `after/afterEach` hook is deprecated and will be removed from a future version of Mocha. Use `return` instead to exit hook abortively.
```

Comment:
Replace this with the language I mentioned in `lib/runner.js`

Reply:
Most used format in Mocha user documentation is `after()` and `afterEach()` hook.

Reply:
ok, "after each" and "after all". done.